database/src/lib.rs

```diff
@@ -17,7 +17,7 @@ pub struct Database {
     pool: Pool
 }
 
-const COUNT_THRESHOLD: u32 = 100000;
+const COUNT_THRESHOLD: u32 = 1000;
```

Comment:
Is the "count" this low by design?

Reply:
Yes, it's too slow otherwise :) So I've lowered this number to increase performance